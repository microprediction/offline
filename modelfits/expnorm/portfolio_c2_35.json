{"g1": 0.05085646369074394, "g2": 14.696148845738337, "logK": -0.6187358265317893, "loc": 0.09299151585040709, "logScale": 0.6786938224827606}