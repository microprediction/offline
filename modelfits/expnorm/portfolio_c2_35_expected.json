{"g1": 0.7733092766583499, "g2": 4.957336180279214, "logK": 0.9942468087133286, "loc": -0.1354228913199802, "logScale": -2.0000883008442143}